store/modules/actions/payment.js

```diff
@@ -49,7 +50,7 @@ export async function sendCosmosPayment(
         memo,
       }, signer));
     }
-    if (metadata) await api.apiPostTxMetadata(txHash, metadata);
+    if (metadata && !isNonLikerKeplrConnected) await api.apiPostTxMetadata(txHash, metadata);
```

Comment:
Note: Maybe we should allow submitting metadata via a signature from the sender address later